ImperatorToCK3/ImperatorToCK3.vcxproj

```diff
@@ -50,6 +50,7 @@
     <PlatformToolset>v142</PlatformToolset>
     <WholeProgramOptimization>false</WholeProgramOptimization>
     <CharacterSet>Unicode</CharacterSet>
+    <EnableASAN>true</EnableASAN>
```

Comment:
address sanitizer is good for checking, but should not be included in released builds as it uses more memory than other builds